djxml/xmlmodels/exceptions.py

```diff
@@ -1,4 +1,6 @@
+from __future__ import absolute_import
 from django.core.exceptions import ValidationError
+import six
```

Comment:
Looks like `six` needs to be added to setup.py `install_requires`

Reply:
(or we could import it from `django.utils`)

Reply:
I've switched to `django.utils`, and have added a note in the README about Django/Python support plan